app/scripts/lib/ppom/ppom-util.ts

```diff
@@ -171,6 +173,7 @@ function normalizePPOMRequest(
       request,
     )
   ) {
+    sanitizeRequest(request);
```

Comment:
rather than updating the original request, can we return a new object? We can update sanitizeRequest to be an idempotent function

Reply:
Good call, done